projects/angular4-paystack/src/lib/angular4-paystack-embed.component.ts

```diff
@@ -10,7 +10,8 @@ declare var window: Partial<MyWindow>;
 @Component({
   selector: 'angular4-paystack-embed',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  template: `<div id="paystackEmbedContainer"></div>`
+  template: `<div id="paystackEmbedContainer"></div>`,
+  standalone: true
```

Comment:
The library support angular from v8-17 (now 19). This would cause older projects to break

Reply:
I've tested changes with angular 16 and 17, and it seems to work pretty well.

Reply:
We should take this out